docs/training/apis-howto/api-keras-ug.rst

```diff
@@ -2,6 +2,9 @@
  Keras API
 ###########
 
+.. meta::
+   :description: Learn how to use the Keras API to train a Keras model. This user guide walks you through loading your data, defining the model, customizing how the model.fit function is called, checkpointing, and callbacks.
```

Comment:
Can `model.fit` but put in double-tildes? I don't know if that's possible or if it matters in a `.. meta::` block.

Reply:
the .. meta:: directive doesn't support inline formatting and Google wouldn't display it even if we used OpenGraph to format it